apps/meteor/tests/unit/app/utils/lib/templateVarHandler.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai';
+import proxyquire from 'proxyquire';
+
+const { templateVarHandler } = proxyquire.noCallThru().load('../../../../../app/utils/lib/templateVarHandler', {
+  'meteor/meteor': {
+    logger: {
+      debug: () => {},
+    },
+    '@global': true,
+  },
+});
+
+const testCases = (description: string, input: { variable: string; object: Record<string, any> }, expectedOutput: string | undefined) => {
+  describe(description, () => {
+    it('should process the variable correctly', () => {
+      const result = templateVarHandler(input.variable, input.object);
+      expect(result).to.be.equal(expectedOutput);
+    });
+  });
+};
+
+describe('templateVarHandler', () => {
```

Comment:
What do you think of, instead of creating a new describe for each item, you construct an array of "test cases" and then you loop on them to do the test?

Something like:

```
const tests = [{ variable: 'attribute1', object: { attribute1: 'value1' }, expected: 'value1' }]

describe('templateVarHandler', () => {
tests.forEach((t) => { it(`should return ${t.expected} when ${t.variable} is passed as key`, () => { expect(templateVarHandler(t.variable, t.object)).to.be.equal(t.expected) }
})
```

(I may have missed some () but wdyt?